framework/decode/vulkan_device_address_tracker.h

```diff
@@ -90,6 +90,8 @@ class VulkanDeviceAddressTracker
     [[nodiscard]] const VulkanAccelerationStructureKHRInfo*
     GetAccelerationStructureByCaptureDeviceAddress(VkDeviceAddress capture_address) const;
 
+    std::unordered_map<VkDeviceAddress, VkDeviceAddress> GetAccelerationStructureDeviceAddressMap() const;
```

Comment:
Missing docs?

Reply:
yes! the function was added adhoc during this PR, added the missing doc

Reply:
And `[[nodiscard]]` might be useful as well :)